dbk-stylebook/lib/transform.js

```diff
@@ -0,0 +1,38 @@
+//Commenting this out so it doesn't create a new file everytime the site is run
```

Comment:
you don't have to worry about this, the script won't run unless we manually trigger it in the console, let's leave it uncommented